File: 1051. Height Checker/solution.cpp
class Solution {
public:
  int heightChecker(vector<int>& heights) {
    vector<int> sortedHeights = heights;
    sort(sortedHeights.begin(), sortedHeights.end());
    int result = 0;
    for (int index = 0; index < heights.size(); index++){
      if (sortedHeights[index] != heights[index]){
        result += 1;
      }
    }

    return result;
  }
};

File: 1051. Height Checker/solution.py
class Solution(object):
  def heightChecker(self, heights):
    """
    :type heights: List[int]
    :rtype: int
    """
    sortedHeights = sorted(heights)
    result = 0
    for index in range(len(heights)):
      if sortedHeights[index] != heights[index]:
        result += 1
    return result